R/extract_soil_map_data.R

```diff
@@ -0,0 +1,62 @@
+#' @title Extract soil properties from Flemish soil map
+#'
+#' @description This function queries the Flemish soil map attributres at a given coordinate,
+#' http://www.geopunt.be/catalogus/datasetfolder/5c129f2d-4498-4bc3-8860-01cb2d513f8f
```

Comment:
I don't think this URL (and the one on line 7) will be clickable in this way. Check https://roxygen2.r-lib.org/articles/formatting.html#links